components/table/index.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import { Table, TextField } from '@oacore/design'
+
+import TablePage from './TablePage'
+import LoadingRow from './LoadingRow'
+import { range } from '../../utils/helpers'
+import tableClassNames from './index.css'
+
+class InfiniteTable extends React.Component {
```

Comment:
This component does too much. Data loading should be part of Model, this it one should be presentational one that fires an action (load more) by either button click or scroll intersection.

_Just a comment, there is nothing to change now._

Reply:
Yes, it just a kick-off, I agree with you it should be handled better. Let's wait for the endpoint to be ready and then we can do appropriate changes.